### PR TITLE
metrics: fix the TSO Grafana panels

### LINKED ISSUE #29940: `PD TSO Wait Duration` on the Grafana panel is ambiguous
### 1. Minimal reproduce step (Required)

`PD TSO Wait Duration` on the `overview.json`.

截屏2021-11-19 17 16 53

https://github.com/pingcap/tidb/blob/177be48e9483db48f5da7f5f2a8ab67368c923cf/metrics/grafana/overview.json#L2731-L2754

`PD TSO Wait Duration` on the `tidb_summary.json`.

截屏2021-11-19 17 17 31

https://github.com/pingcap/tidb/blob/177be48e9483db48f5da7f5f2a8ab67368c923cf/metrics/grafana/tidb_summary.json#L2687-L2710

`PD TSO Wait Duration` on the `tidb.json`.

截屏2021-11-19 17 17 09

https://github.com/pingcap/tidb/blob/177be48e9483db48f5da7f5f2a8ab67368c923cf/metrics/grafana/tidb.json#L8347-L8370

### 2. What did you expect to see? (Required)

They should all be the same.

### 3. What did you see instead (Required)

`PD TSO Wait Duration` on the `overview.json` is not the same as the other two.

### 4. What is your TiDB version? (Required)

177be48e9483db48f5da7f5f2a8ab67368c923cf

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Issue Number: close #29940.

Problem Summary:

Fix the TSO Grafana panels.

### What is changed and how it works?

* Make `PD TSO Wait Duration` on the `overview.json` is the same as the other two.
* Change `Start TSO Wait Duration` to `Async TSO Duration`.

According to https://github.com/tikv/client-go/blob/b11da6307c6fc761e0d8841a72bccd578be1ef82/oracle/oracles/pd.go#L117, `Start TSO Wait Duration` is duplicated with `PD TSO Wait Duration` actually. `Async TSO Duration` is added to show the duration of an Async TSO until called the `Wait` function.

image

### Check List

Tests 

- [x] No code

### Release note

```release-note
Fix the inconsistency of TiDB TSO-related Grafana panels.
```

### CODE REVIEW

File: metrics/grafana/tidb.json [hunk 1]
```diff
@@ -8515,111 +8515,6 @@
...
-              "intervalFactor": 2,
-              "legendFormat": "90",
-              "refId": "C"
-            }
-          ],
-          "thresholds": [],
-          "timeFrom": null,
-          "timeRegions": [],
-          "timeShift": null,
-          "title": "Start TSO Wait Duration",
```
Comment (nolouch) [inline metrics/grafana/tidb.json:8585]: Can we add `Async Wait`?

**Review by disksing — APPROVED** [review]

File: metrics/grafana/tidb.json (hunk 1 above)
Comment (JmPotato) [reply metrics/grafana/tidb.json:8585]: Done. PTAL.

**Review by disksing — APPROVED** [review]

File: metrics/grafana/tidb.json [hunk 2]
```diff
@@ -8556,22 +8556,22 @@
...
-              "expr": "histogram_quantile(0.99, sum(rate(tidb_pdclient_ts_future_wait_seconds_bucket{tidb_cluster=\"$tidb_cluster\"}[1m])) by (le))",
+              "expr": "histogram_quantile(0.99, sum(rate(pd_client_cmd_handle_cmds_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\", type=\"tso\"}[1m])) by (le))",
               "format": "time_series",
               "intervalFactor": 2,
               "legendFormat": "99",
               "refId": "B"
             },
             {
-              "expr": "histogram_quantile(0.90, sum(rate(tidb_pdclient_ts_future_wait_seconds_bucket{tidb_cluster=\"$tidb_cluster\"}[1m])) by (le))",
+              "expr": "histogram_quantile(0.90, sum(rate(pd_client_cmd_handle_cmds_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\", type=\"tso\"}[1m])) by (le))",
```
Comment (nolouch) [inline metrics/grafana/tidb.json:8574]:
```suggestion
              "expr": "histogram_quantile(0.90, sum(rate(pd_client_cmd_handle_cmds_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\", type=\"tso_async_wait\"}[1m])) by (le))",
```

File: metrics/grafana/tidb.json [hunk 3]
```diff
@@ -8556,22 +8556,22 @@
...
+              "expr": "histogram_quantile(0.999, sum(rate(pd_client_cmd_handle_cmds_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\", type=\"tso\"}[1m])) by (le))",
               "format": "time_series",
               "intervalFactor": 2,
               "legendFormat": "999",
               "refId": "A",
               "step": 10
             },
             {
-              "expr": "histogram_quantile(0.99, sum(rate(tidb_pdclient_ts_future_wait_seconds_bucket{tidb_cluster=\"$tidb_cluster\"}[1m])) by (le))",
+              "expr": "histogram_quantile(0.99, sum(rate(pd_client_cmd_handle_cmds_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\", type=\"tso\"}[1m])) by (le))",
```
Comment (nolouch) [inline metrics/grafana/tidb.json:8567]:
```suggestion
              "expr": "histogram_quantile(0.99, sum(rate(pd_client_cmd_handle_cmds_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\", type=\"tso_async_wait\"}[1m])) by (le))",
```

**Review by nolouch — APPROVED** [review]

**Pull request merged**